apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2Test.java

```diff
@@ -66,225 +77,296 @@ public void setup() {
             .collect(Collectors.toSet());
     beneficiary.getMedicareBeneficiaryIdHistories().addAll(beneficiaryMbis);
     assertThat(beneficiary, is(notNullValue()));
+
+    createPatient(RequestHeaders.getHeaderWrapper());
+  }
+
+  private void createPatient(RequestHeaders reqHeaders) {
+    patient = BeneficiaryTransformerV2.transform(new MetricRegistry(), beneficiary, reqHeaders);
   }
 
   @After
   public void tearDown() {
     beneficiary = null;
+    patient = null;
   }
```

Comment:
Is this `After` function needed? Like what's the consequence for not setting these variables back to null?

Reply:
I thought I had replied to this, but I guess not; just being anal and making sure state/data is cleared after every test.